packages/polyfills/types/global.d.ts

```diff
@@ -1 +1,12 @@
 declare function r2d2Fetch(url: RequestInfo, init?: RequestInit): Promise<Response>
+
+interface GlobalEvent<T> extends Event {
+    payload: T
+}
+
+interface WindowEventMap {
+    scenechange: GlobalEvent<{
+        scene: string
```

Comment:
Maybe we will have more scenes in the future.

```suggestion
        scene: SceneType
```